src/mesh/NodeDB.cpp

```diff
@@ -316,8 +316,8 @@ void NodeDB::installDefaultChannels()
 
 void NodeDB::resetNodes()
 {
-    devicestate.node_db_lite_count = 0;
-    memset(devicestate.node_db_lite, 0, sizeof(devicestate.node_db_lite));
+    devicestate.node_db_lite_count = 1;
+    std::fill(&devicestate.node_db_lite[1], &devicestate.node_db_lite[MAX_NUM_NODES - 1], meshtastic_NodeInfoLite());
```

Comment:
Is the current device's nodeinfo guaranteed to be in this index?

Reply:
It's the first entry generated after the nodedb is initialized and I don't believe there are any functions or processes that would cause the ordering of nodes in the database to be re-sorted or re-arranged. To the best of my knowledge it should remain as the first entry in the nodedb.